### PR TITLE
[#ISV-29] Enable Mit Voucher Jwt Token

### PR DESCRIPTION
**Pull request by AleDore**
This PR enables Mit Voucher JWT generation

### CODE REVIEW

File: src/services/tokenService.ts [hunk 1]
```diff
@@ -58,4 +58,35 @@ export default class TokenService {
       )
     )().mapLeft(toError);
   }
+
+  /**
+   * Generates a new support token containing the logged user's fiscalCode.
```
Comment (AleDore) [inline src/services/tokenService.ts:63]:
```suggestion
   * Generates a new bit voucher token containing the logged user's fiscalCode.
```

File: src/services/tokenService.ts [hunk 2]
```diff
@@ -58,4 +58,35 @@ export default class TokenService {
...
+   *
+   * @param privateKey: The RSA's private key used to sign this JWT token
+   * @param fiscalCode: The logged user's FiscalCode
+   * @param tokenTtl: Token Time To live (expressed in seconds)
+   * @param issuer: The Token issuer
```
Comment (AleDore) [inline src/services/tokenService.ts:68]:
```suggestion
   * @param issuer: The Token issuer
   * @param audience: The Token audience
```

**Comment by AleDore** [comment]
@iwoak : It seems that unit tests fail :)

**Comment by iwoak** [comment]
@AleDore fixed it :|

**Pull request merged**